specification/storagesync/resource-manager/Microsoft.StorageSync/2017-06-05-preview/storagesync.json

```diff
@@ -0,0 +1,3232 @@
+          "200": {
+            "description": "Storage Sync Service object was deleted.",
+            "headers": {
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Storage Sync Service Resource doesn't exist"
+          },
+          "default": {
+            "description": "Error message indicating why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/storageSyncServices": {
+      "get": {
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "StorageSyncServices Resource"
+        ],
+        "operationId": "StorageSyncServiceByResourceGroupGet_list",
+        "description": "Get a StorageSyncService list by Resource group name.",
+        "x-ms-examples": {
+          "StorageSyncServiceGet_ListByResourceGroup": {
+            "$ref": "./examples/StorageSyncServiceGet_ListByResourceGroup.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Array of registered Storage Sync Service resources in the Resource Group",
+            "schema": {
+              "$ref": "#/definitions/StorageSyncServiceArray"
+            },
+            "headers": {
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error message indicating why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageSync/storageSyncServices": {
+      "get": {
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "StorageSyncServices Resource"
+        ],
+        "operationId": "StorageSyncServiceGet_ListBySubscription",

```

Comment:
Operation ID should be in the format `"group_verb"`. So this should be changed to something like:
`"operationId": "StorageSyncServices_ListBySubscription"`

Reply:
resolved